server/serializer/subscription.go

```diff
@@ -179,3 +182,85 @@ func SubscriptionFromJSON(data io.Reader) (*SubscriptionPayload, error) {
 
 	return sp, nil
 }
+
+func GetFormattedSubscriptionEvents(subscriptionEvents string) string {
```

Comment:
You can also use this function inside `GetFormattedSubscriptions` function. Also, take a reference from there for using `strings.Builder`.